scripts/who/gho/test_data/expected_csv.csv

```diff
@@ -0,0 +1,434 @@
+year,period,country,statVar,value
+2015,P1Y,country/ARG,dcs:WHO/EMFLIMITELECTRIC_Public_LowFrequency,3
```

Comment:
While this does work, I think the its more appropriate to use a reference to the country, as "dcid:country/ARG".

Reply:
done